src/Core/Tools/ImportFeatures/ImportCiphersCommand.cs

```diff
@@ -26,7 +29,8 @@ public class ImportCiphersCommand : IImportCiphersCommand
     private readonly ICollectionRepository _collectionRepository;
     private readonly IReferenceEventService _referenceEventService;
     private readonly ICurrentContext _currentContext;
-
+    private readonly IPolicyRequirementQuery _policyRequirementQuery;
+    private readonly IFeatureService _featureService;
 
     public ImportCiphersCommand(
```

Comment:
🌱 This constructor now has 10 dependencies. That's a good indication that this code is doing too much and a refactoring is in order. 

❓ What would you recommend changing if you could? Dream big!

Reply:
Excellent observation! I think an easy thing to do would be to separate the individual vault import and the org vault import each into its own command. From there we could extract the validation logic into a validation class.

Reply:
@djsmith85 ☝🏻